src/cockpit/bridge.py

```diff
@@ -148,7 +148,7 @@ def do_send_init(self) -> None:
     # PackagesListener interface
     def packages_loaded(self):
```

Comment:
Can you add a `-> None` as the return type here and clean up the mypy stuff?  No signature on a function means that mypy doesn't check its body for correctness.  I think mypy would have caught this error if it had a chance...

Reply:
Done in a separate commit. mypy was still happy even with this, as BridgeConfig is a JsonObject, presumaly?

Reply:
Nevermind -- mypy wasn't running as my worktree was missing modules. Now it triggers

> src/cockpit/bridge.py:150: error: Item "None" of "Packages | None" has no attribute "get_bridge_configs"  [union-attr]

I'll look into that.

Reply:
This sounds like a good place for an `assert`.